egs2/TEMPLATE/asr2/asr2.sh

```diff
@@ -1304,15 +1327,19 @@ if [ ${stage} -le 13 ] && [ ${stop_stage} -ge 13 ] && ! [[ " ${skip_stages} " =~
             log "${_split_dir}/.done exists. Spliting is skipped"
         fi
 
-        _opts+="--train_data_path_and_name_and_type ${_split_dir}/text.${src_case}.${src_lang},src_text,text "
-        _opts+="--train_data_path_and_name_and_type ${_split_dir}/text.${tgt_case}.${tgt_lang},text,text "
+        _opts+="--train_data_path_and_name_and_type ${_split_dir}/text.${src_case}.${src_lang},src_text,text " 
+        _opts+="--train_data_path_and_name_and_type ${_split_dir}/text.${tgt_case}.${tgt_lang},text,text " 
         _opts+="--train_shape_file ${_split_dir}/src_text_shape.${src_token_type} "
         _opts+="--train_shape_file ${_split_dir}/text_shape.${tgt_token_type} "
         _opts+="--multiple_iterator true "
     else
-        _opts+="--train_data_path_and_name_and_type ${_asr_train_dir}/text.${src_case}.${src_lang},src_text,text "
-        _opts+="--train_data_path_and_name_and_type ${_asr_train_dir}/text.${tgt_case}.${tgt_lang},text,text "
-        _opts+="--train_shape_file ${asr_stats_dir}/train/src_text_shape.${src_token_type} "
+        _opts+="--train_data_path_and_name_and_type ${_asr_train_dir}/text.${src_case}.mfcc_km10,src_text,text "
+        _opts+="--train_data_path_and_name_and_type ${_asr_train_dir}/text.${src_case}.${src_lang},src_text1,text1 "
+        _opts+="--train_data_path_and_name_and_type ${_asr_train_dir}/text.${tgt_case}.${tgt_lang},text,text " 
+        _opts+="--train_data_path_and_name_and_type ${_asr_train_dir}/text.${tgt_case}.${tgt_lang},text1,text1 " 
+        _opts+="--train_shape_file ${asr_stats_dir}/train/src_text_shape.${src_token_type} " 
+        _opts+="--train_shape_file ${asr_stats_dir}/train/src_text_shape.${src_token_type} " 
+        _opts+="--train_shape_file ${asr_stats_dir}/train/text_shape.${tgt_token_type} "
```

Comment:
I do not think you need a text1 here, as the prediction is supposed to be the same (as original text)